kfdef/kfctl_gcp_iap.yaml

```diff
@@ -1,17 +1,10 @@
-# Please set project and email!
-apiVersion: kfdef.apps.kubeflow.org/v1alpha1
+apiVersion: kfdef.apps.kubeflow.org/v1beta1
 kind: KfDef
 metadata:
   creationTimestamp: null
   name: myapp2
```

Comment:
Remove name from KFDef files so that its auto filled based on directory.

Reply:
commented out.